core/src/main/java/org/mobilitydata/gtfsvalidator/notice/UnexpectedEnumValueError.java

```diff
@@ -18,7 +18,11 @@
 
 import com.google.common.collect.ImmutableMap;
 
-/** An enum has an unexpected value. */
+/**
+ * An enum has an unexpected value.
+ *
+ * <p>Severity: {@code SeverityLevel.ERROR}
```

Comment:
I think this has to be a WARNING? There are a lot of examples where producers have extended enumeration values in their own feeds with values that don't exist in the spec. And to my knowledge the spec doesn't explicitly say you can't use non-standard enum values.